recipes/dasel/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "dasel" %}
+{% set author = "TomWright" %}
+{% set version = "1.24.3" %}
+{% set pkg_src = 'src/' + name %}
```

Comment:
Please remove the jinja variables for all except version. Jinja variables are for things that need to change frequently. i.e. the version number, but the package name and directories should be relatively constant throughout the life of the recipe. Additionally, certain metadata is extracted by conda-forge from the about section without running Jinja.

Reply:
Ok, thanks! This part was easy.

Now about the licenses........ I need to figure out what must be done there.